app/models/portal/offering.rb

```diff
@@ -118,7 +118,9 @@ def reportable?
     if internal_report?
       true
     else
-      runnable.class == ExternalActivity && !runnable.report_url.blank?
+      # runnable.class == ExternalActivity && !runnable.report_url.blank?
+      # We no longer handle report_url in external activities, see 3f7638a
+      false
```

Comment:
Since report_url will be gone from the code I think this will be confusing.  I'd just changed it to return `internal_report?`.  We could simplify this even more, by consolidating all of the methods that just return `internal_report?`.  But that would touch more code so it seems good to skip that for now.